examples/kitchen-sink.yaml

```diff
@@ -249,8 +249,11 @@ meter_provider:
             record_min_max: true
         # Configure attribute keys retained in the resulting stream(s).
         attribute_keys:
-          - key1
-          - key2
+          # Configure list of attribute key patterns that are retained in the resulting stream(s).
+          # All other attributes not matching will be dropped.
+          included:
```

Comment:
The expansion from absolute values to patterns extends beyond the language of the spec. My understanding is that this adjustment is to help improve ergonomics for when the spec also includes the ability to exclude values. Can we start with absolute keys and evolve to allow patterns if the spec adds support?

Reply:
Done